base/applications/cmdutils/more/lang/zh-CN.rc

```diff
@@ -1,54 +1,53 @@
 
 LANGUAGE LANG_CHINESE, SUBLANG_CHINESE_SIMPLIFIED
 
 STRINGTABLE
 BEGIN
     IDS_USAGE "在屏幕上一页一页地显示内容。\n\
 \n\
-MORE [/E [options]] < [drive:][path]filename\n\
-command | MORE [/E [options]]\n\
+MORE [/E [选项]] < [盘符:][路径]文件名\n\
+命令 | MORE [/E [选项]]\n\
 \n\
-    [drive:][path]filename  A file whose contents are to be displayed.\n\
-    command                 A command whose output is to be displayed.\n\
+    [盘符:][路径]文件名  要显示内容的文件的路径。\n\
+    命令                  要显示输出的命令。\n\
 \n\
-When Extended Features are enabled, the following form is supported:\n\
-MORE [/E] [options] [files]\n\
+当启用扩展特性之后，可以使用以下形式的命令：\n\
+MORE [/E] [选项] [文件]\n\
 \n\
-Options:\n\
-    /E       Enable Extended Features.\n\
-    The default state of the Extended Features is controlled by the default\n\
-    registry settings for CMD.EXE. For more details on how to change these\n\
-    settings, see the section ""Command Extensions"" in CMD.EXE help.\n\
+选项：\n\
+    /E       启用扩展特性。\n\
+    扩展特性默认启用与否是由 cmd.exe 的默认注册表设置控制的。\n\
+    关于如何改变这些设置的详细信息，请参见 cmd.exe 帮助内容的“命令扩展”一节。\n\
 \n\
-    When Extended Features are enabled, the following options are available:\n\
-    /C       Clear the screen before displaying a page.\n\
-    /P       Expand Form-Feed characters.\n\
-    /S       Shrink successive blank lines to a single line.\n\
-    /Tn      Expand tabs to n spaces (default: 8).\n\
+    当启用扩展特性之后，可以使用以下形式的选项：\n\
+    /C       在显示下一页之前先清屏。\n\
+    /P       展开分页符。\n\
+    /S       将连续的空行合并为一个空行。\n\
+    /Tn      指定将制表符扩展为几个空格（默认：8）。\n\
 \n\
-    These options can be specified in the MORE environment variable.\n\
+    可以在 MORE 环境变量中指定以下选项：\n\
 \n\
-    +n       Start displaying the first file at line n.\n\
-    files    List of files to display, separated by blanks.\n\
+    +n       从第 n 行开始显示第一个文件。\n\
+    files    要显示内容的文件的列表，文件之间以空白符分隔。\n\
 \n\
-When Extended Features are enabled, the following commands are available\n\
-at the ""-- Continue --"" prompt:\n\
+当启用扩展特性之后，可以在“-- 继续 --”提示符处使用以下命令：\n\
 \n\
-    P n      Display the next n lines.\n\
-    S n      Skip the next n lines.\n\
-    F        Display the next file.\n\
-    Q        Quit.\n\
-    =        Show the line number.\n\
-    ?        Show the help line.\n\
-    <Space>  Display the next page.\n\
-    <Enter>  Display the next line.\n"
+    P n      显示下面的 n 行。\n\
+    S n      跳过 n 行。\n\
+    F        开始显示下一个文件。\n\
+    Q        退出。\n\
+    =        显示行号。\n\
+    ?        显示帮助提示符。\n\
+    <空格>   显示下一页。\n\
+    <回车>   显示下一行。\n"
 
-    IDS_BAD_FLAG "Invalid argument - '%s'\n"
+    IDS_BAD_FLAG "非法参数 - '%s'\n"
     IDS_FILE_ACCESS "无法访问文件 %s。\n"
     IDS_CONTINUE_PROMPT  "-- 继续%1%2%3 --%4"
     IDS_CONTINUE_PERCENT " (%d%%)"
-    IDS_CONTINUE_LINE_AT " [Line: %d]"
-    IDS_CONTINUE_OPTIONS " [Options: psfq=<Space><Enter>]"
-    IDS_CONTINUE_LINES   " Lines: "
+    IDS_CONTINUE_LINE_AT " [第 %d 行]"
+    IDS_CONTINUE_OPTIONS " [选项：psfq=<空格><回车>]"
+    IDS_CONTINUE_LINES   " 行数: "
```

Comment:
```suggestion
    IDS_CONTINUE_LINES   " 行数："
```

Reply:
I somehow left this colon change uncommitted 🤔